ci/vars-zuul-common.yml

```diff
@@ -1,5 +1,5 @@
 ---
-namespace: "service-telemetry-PR#{{ zuul.change }}-{{ zuul.build }}"
+namespace: "service-telemetry"
```

Comment:
Can you not just remove the # part and add a `| lower` function here? I'd still rather not lose the non-default namespace test.

Reply:
I'll reluctantly approve this to get things working, but can you file a follow up task to come back and make sure we can test with dynamic namespaces?

Reply:
I've created an issue to track this internally